timeseries/src/autogluon/timeseries/predictor.py

```diff
@@ -893,6 +893,120 @@ def evaluate(
             logger.info(json.dumps(scores_dict, indent=4))
         return scores_dict
 
+    def feature_importance(
+        self,
+        data: Optional[Union[TimeSeriesDataFrame, pd.DataFrame, Path, str]],
```

Comment:
Defaults to `None` (same as in `TabularPredictor.feature_importance` and `TimeSeriesPredictor.leaderboard`)?